extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -28,14 +28,13 @@ import { ImportDataModel } from '../models/api/import';
 import { NetCoreTool, DotNetCommandOptions } from '../tools/netcoreTool';
 import { BuildHelper } from '../tools/buildHelper';
 
-// TODO: use string enums
 export enum ExtractTarget {
-	dacpac = 0,
-	file = 1,
-	flat = 2,
-	objectType = 3,
-	schema = 4,
-	schemaObjectType = 5
+	dacpac = 'dacpac',
```

Comment:
Just a nit: Though we aren't using "dacpac" anywhere, but better would be to match the case with other enum values.

Reply:
removed it since the enum got removed